website/docs/components/dropdown/partials/code/component-api.md

```diff
@@ -41,6 +59,18 @@ The Dropdown component is composed of different child components each with their
   </C.Property>
   <C.Property @name="color" @type="enum" @values={{array "primary" "secondary" }} @default="primary"/>
   <C.Property @name="size" @type="enum" @values={{array "medium" "small" }} @default="medium"/>
+  <C.Property @name="icon" @type="string">
+    Acceptable value: any [icon](/icons/library) name.
```

Comment:
```suggestion
    Used to show an icon. Any [icon](/icons/library) name is accepted.
```

Suggestion based on prior art: https://github.com/hashicorp/design-system/pull/1261/files and looking at other component docs files. 

But there are a lot of components that we may want to go through and align on this, so it's a non-blocking suggestion.